node-notes-cli/app.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+
+const arg1 = process.argv[2];
+
+if (arg1 === 'read') {
+  fs.readFile('./data.json', 'utf8', (err, data) => {
+    if (err) {
+      throw err;
+    }
+    console.log(data);
+  });
+} else if (arg1 === 'create') {
+  fs.readFile('./data.json', 'utf8', (err, data) => {
+    if (err) {
+      throw err;
+    }
+    const arg2 = process.argv[3];
```

Comment:
You should use more meaningful variable names. For example here a better name would be something like `note`

Reply:
hmm fair enough